languages.json

```diff
@@ -1946,6 +1946,12 @@
       "blank": true,
       "extensions": ["sln"]
     },
+    "VisualStudioSolutionSlnx": {
+      "name": "Visual Studio Solution",
```

Comment:
I'm not sure what happens if you run this in a folder with both a `.sln` and `.slnx` file in the same directory structure. Should this one maybe have a name such as "Visual Studio Solution (slnx)"?

Reply:
Thank you for your PR! I believe it will just show two rows with the same name. I think having something to distinguish it like what you mentioned would be a good idea.

Reply:
I have updated the name to "Visual Studio Solution (slnx)".